src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -251,9 +251,12 @@ public void rm(@Nonnull EnvVars launchEnv, @Nonnull String containerId) throws I
         LaunchResult result = launch(new EnvVars(), true, "-v");
```

Comment:
Why just not use the `--version` option? It is supported by both *docker* (see https://docs.docker.com/engine/reference/commandline/cli/) and *podman*.

```suggestion
        LaunchResult result = launch(new EnvVars(), true, "--version");
```

This way, there is no need to add another if-statement if the `-v` option fails.